packages/next/src/lib/server-external-packages.json

```diff
@@ -24,5 +24,6 @@
   "ts-node",
   "typescript",
   "vscode-oniguruma",
-  "webpack"
+  "webpack",
+  "bcrypt"
```

Comment:
maybe sort this alphabetically like the others?

Reply:
> maybe sort this alphabetically like the others?

Fair, I'll do that right away 👍